tests/log-in.test.js

```diff
@@ -0,0 +1,75 @@
+//const { default: $ } = require("webdriverio/build/commands/browser/$");
+describe('Banking', () => {
+
+    it('Debería ingresar a la pagina con datos correctos', async () => {
+        await browser.url('login');
+
+        let userName = await $('#username');
+        await userName.setValue('jsmith@demo.io');
+
+        let pssWord = await $('#password');
+        await pssWord.setValue('Demo123!');
+
+        let submit = await $('#submit');
+        await submit.click();
+
+        let url = await browser.getUrl(); 
+        //await console.log(url);
+        //await console.log("salida");
```

Comment:
Commented code is an instance of **dead code**. Remove it. With git, there's no need to keep it (you can go back in time at any time to check how code was back then)